### PR TITLE
KAA-1020: Fix Log Collector unit tests.

### PR DESCRIPTION
**Pull request by ohamov**

### CODE REVIEW

**Comment by kaa-leeroy** [comment]
Can one of the admins verify this patch?

**Comment by rasendubi** [comment]
@kaa-leeroy add to whitelist

**Comment by forGGe** [comment]
Looks good

**Comment by GamovCoder** [comment]
@dyosick

**Comment by rasendubi** [comment]
LGTM

File: client/client-multi/client-cpp/test/impl/log/LogCollectorTest.cpp [hunk 1]
```diff
@@ -81,6 +81,17 @@ static LogRecord createSerializedLogRecord()
     return LogRecord(logRecord);
 }
 
+static void joinThreadsSync(SimpleExecutorContext& executor)
+{
+    auto& lifeCycleExecutor = executor.getLifeCycleExecutor();
+    auto& apiExecutor = executor.getApiExecutor();
+    auto& callbackExecutor = executor.getCallbackExecutor();
+
+    dynamic_cast<ThreadPool&>(lifeCycleExecutor).shutdownNow();
```
Comment (ghost) [inline client/client-multi/client-cpp/test/impl/log/LogCollectorTest.cpp:90]: Why do your cast? The IThreadPool interface already has the shutdownNow method.

Comment (ohamov) [inline client/client-multi/client-cpp/test/impl/log/LogCollectorTest.cpp:90]: Yep, somehow missed it, thanks

**Pull request merged**